src/qibolab/instruments/qblox/sequencer.py

```diff
@@ -5,12 +5,22 @@
 
 from qibolab.instruments.qblox.q1asm import Program
 from qibolab.pulses import Pulse, PulseSequence, PulseType
+from qibolab.pulses.shape import modulate
 from qibolab.sweeper import Parameter, Sweeper
 
 SAMPLING_RATE = 1
 """Sampling rate for qblox instruments in GSps."""
 
 
+def _modulated_waveforms(pulse: Pulse, hardware: bool = True):
```

Comment:
I would prefer to have a better name for this function, since it doesn't always return modulated waveforms as the name suggests. But this is very minor thing, so I am fine either way.

Reply:
I was also tempted to name it
```suggestion
def _maybe_modulated_waveforms(pulse: Pulse, hardware: bool = True):
```
or even just
```suggestion
def _maybe_modulated(pulse: Pulse, hardware: bool = True):
```

In the end, I decided for the simpler (though ambiguous) option.

Reply:
In lack of any better proposal, I will just keep the current one.